docs/source/tutorials-basic/led.rst

```diff
@@ -29,7 +29,7 @@ The code we're analyzing in this tutorial is the following one.
 
 
     # create the LED instance, passing the port and GPG
-    my_led = gpg.init_led("AD1")
+    my_led = gpp.init_led("AD1")
```

Comment:
this typo is back?

Reply:
I'm not sure. Something happened from this morning and now everything's haywire. I don't want to make a new branch though.

Literally, there were 4-6 files changed this morning and now there are a whopping 57 files. Weird.